test/hw2/test_hw3.py

```diff
@@ -0,0 +1,131 @@
+"""TEST Homework 2.3."""
+
+from typing import Any, List
+
+import pytest
+
+from hw.hw2.hw3 import combinations
+
+
+@pytest.mark.parametrize(
+    ("expected_result", "value"),
+    [
+        (
+            [
+                [1, 4],
+                [1, 5],
+                [2, 4],
+                [2, 5],
+                [3, 4],
+                [3, 5],
+                [4, 6],
+                [4, 7],
+                [5, 6],
+                [5, 7],
+                [6, 8],
+                [6, 9],
+                [6, 10],
+                [7, 8],
+                [7, 9],
+                [7, 10],
+                [8, 11],
+                [8, 12],
+                [8, 13],
+                [9, 11],
+                [9, 12],
+                [9, 13],
+                [10, 11],
+                [10, 12],
+                [10, 13],
+            ],
+            ([1, 2, 3], [4, 5], [6, 7], [8, 9, 10], [11, 12, 13]),
+        ),
+        (
+            [
+                [1, 4],
+                [1, 5],
+                ["2", 4],
+                ["2", 5],
+                [3, 4],
+                [3, 5],
+                [4, None],
+                [4, 7],
+                [5, None],
+                [5, 7],
+                [None, 8],
+                [None, 9],
+                [None, 10],
+                [7, 8],
+                [7, 9],
+                [7, 10],
+                [8, 11],
+                [8, 12],
+                [8, 13],
+                [9, 11],
+                [9, 12],
+                [9, 13],
+                [10, 11],
+                [10, 12],
+                [10, 13],
+            ],
+            ([1, "2", 3], [4, 5], [None, 7], [8, 9, 10], [11, 12, 13]),
```

Comment:
Are these examples of combinations correct? According to example of test:

`assert combinations([1, 2], [3, 4]) == [
    [1, 3],
    [1, 4],
    [2, 3],
    [2, 4],
]`

we should achieve many elements such as [1, 5], [1, 4] ...